samples/net/sockets/net_mgmt/sample.yaml

```diff
@@ -0,0 +1,15 @@
+common:
+  harness: net
+  depends_on: netif
+  tags: net socket mgmt
+sample:
+  description: Test network management socket
+    sample
+  name: net_mgmt socket
+tests:
+  sample.net.sockets.net_mgmt.kernelmode:
+    extra_configs:
+      - CONFIG_USERSPACE=n
```

Comment:
userspace isn't enabled by default

Reply:
Sure. I placed the config there as one cannot leave the test "section" empty, it needs to have some setting in it.